src/igtcloud/client/tools/upload_project.py

```diff
@@ -32,6 +34,32 @@ def upload_project(local_folder: str, project_name: str, institute_name: str = N
     if submit:
         _password = getpass("For electronic record state it is required to reenter the password")
 
+    if category.lower() == "annotation":
+        # Annotation file upload
+        logger.info("Annotation File Uploading...")
+        if os.path.isdir(local_folder):
+            for root, dirs, local_files in os.walk(local_folder):
+                key = root
+                annotation_file = local_files
+        else:
+            logger.error("Not a valid Directory")
+        key = key + "/" + str(annotation_file[0])
+        logger.debug("Upload annotation path" + key)
+        try:
+            f = open(os.path.abspath(key), "r")
+            json.loads(f.read())
```

Comment:
It is good the check whether the annotations are valid JSON, but there is one file in the file hierarchy which is not a JSON file, and that is the file with the annotator's name as file name (in the `name` folder).

Reply:
Ignored the non JSON file and not upload other than JSOn file , Hence the name file will not be uploaded

Reply:
There is one file in the annotation hierarchy with the annotators name as file name (in the `name` folder), e.g. `Robin_Laugs`. This file is not a JSON file, but it **should** be uploaded.

Reply:
Same has been taken care